### DISCUSSION TITLE
v1.4.2

### DISCUSSION THREAD

**Discussion by abarichello** [R2Northstar/Northstar#192] [Release]
Fixed a crash that re-enabled networks when leaving a match.
Stopped checking for Stryder response to unlock 'Launch Northstar' button (loads 5 seconds faster).
Removed logging of server names after doing a server fetch.

Release generated by: https://github.com/R2Northstar/Northstar/actions/runs/1826782500

Thanks:
@emma-miller @GeckoEidechse

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.4.2'>v1.4.2</a>.</em>

**Comment by x3Karma** [comment]
thanks @abarichello very gamer

**Comment by TitiRoc** [comment]
Merci 👍
